src/libratbag-data.c

```diff
@@ -384,7 +392,7 @@ match(const struct input_id *id, char **strv)
 }
 
 static bool
-file_data_matches(struct ratbag *ratbag,
+parse_device_file(struct ratbag *ratbag,
```

Comment:
Keep the name.

Reply:
but its so cryptic :(

Reply:
If you want to change it please do it in a separate PR :)

Reply:
ok :D